CHANGES.rst

```diff
@@ -4,6 +4,12 @@ Changelog for zest.releaser
 4.0 (unreleased)
 ----------------
 
+- Added support for twine_.  If the ``twine`` command is available, it
+  is used for uploading to PyPI.  It is added to the ``recommended``
```

Comment:
Well, the package will be installed then, but the command script might not be installed, or may not be on your path if zest.releaser is not on the path. With `pip` it is fine, but with `zc.buildout` you need to explicitly add `twine` to the eggs, as we do in the `buildout.cfg` in this pull request:

```
[script]
recipe = zc.recipe.egg
interpreter = python
eggs =
    zest.releaser
    zest.releaser[recommended]
    twine
```

Otherwise the twine package is installed, but the twine command not. Do you know a different incantation of this or another recipe that does this better?

I can change the wording a bit.